fedora_messaging/_session.py

```diff
@@ -153,7 +153,7 @@ def publish(self, message, exchange=None):
         except (pika_errs.NackError, pika_errs.UnroutableError) as e:
             _log.warning("Message was rejected by the broker (%s)", str(e))
             raise PublishReturned(reason=e)
-        except (pika_errs.ConnectionClosed, pika_errs.AMQPChannelError):
```

Comment:
Why do we want to stop handling ConnectionClosed here?

Reply:
It's a subclass: https://pika.readthedocs.io/en/stable/_modules/pika/exceptions.html#ConnectionClosed